charts/nats/templates/pod-disruption-budget.yaml

```diff
@@ -0,0 +1,14 @@
+##############################
+## NATS Pod Disruption Budget
+##############################
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Release.Name }}-nats-pdb
+spec:
+  maxUnavailable: 1
+  selector:
+    matchLabels:
+      tier: logging
```

Comment:
The tier here and on the Stan file should be something other than logging.

Reply:
good call https://github.com/astronomer/astronomer/blob/b0abcf6c5f0765d58309ff97bfa07dfe051a5e20/charts/nats/templates/statefulset.yaml#L32

Reply:
Doesn't look like any of the other nats charts have a `tier:` label. So just make this `nats` for both the nat template and stan template since we think of them both as part of the nats layer

Reply:
I realized I do need to add the Houston Worker too.

Reply:
Cool LGTM

Reply:
oops...just saw the next comment.   I'll change from `astronomer` to `nats`.